orm.go

```diff
@@ -1016,16 +1021,27 @@ func formatRowColIDKey(rowIDOrKey, colIDOrKey interface{}) (string, string, erro
 	return rowStr, colStr, err
 }
 
-// FieldType
+// FieldType is the type of a field.
 type FieldType string
 
 const (
+	// FieldTypeDefault is the default field type.
 	FieldTypeDefault FieldType = ""
-	FieldTypeSet     FieldType = "set"
-	FieldTypeInt     FieldType = "int"
-	FieldTypeTime    FieldType = "time"
-	FieldTypeMutex   FieldType = "mutex"
-	FieldTypeBool    FieldType = "bool"
+	// FieldTypeSet is the set field type.
+	// See: https://www.pilosa.com/docs/latest/data-model/#set
+	FieldTypeSet FieldType = "set"
+	// FieldTypeInt is the int field type.
+	// See: https://www.pilosa.com/docs/latest/data-model/#int
+	FieldTypeInt FieldType = "int"
+	// FieldTypeTime is the time field type.
+	// See: https://www.pilosa.com/docs/latest/data-model/#time
+	FieldTypeTime FieldType = "time"
+	// FieldTypeMutex is the mutex field type.
+	// See: https://www.pilosa.com/docs/latest/data-model/#mutex
+	FieldTypeMutex FieldType = "mutex"
+	// FieldTypeBool is the boolean field type.
+	// See: https://www.pilosa.com/docs/latest/data-model/#boolean
```

Comment:
I like the links for the specific field types. Possibly we should also have such a link on the FieldType description?

Reply:
Thanks, added the link.